pkg/broker/user_info.go

```diff
@@ -0,0 +1,78 @@
+package broker
+
+import (
+	"encoding/json"
+	"fmt"
+
+	osb "github.com/pmorie/go-open-service-broker-client/v2"
+)
+
+// parseKubernetesIdentity - creates a kubernetes identity from the
+// orginating identity
+func parseKubernetesIdentity(o osb.OriginatingIdentity) (*UserInfo, error) {
+	u := UserInfo{}
+	err := json.Unmarshal([]byte(o.Value), &u)
+	if err != nil {
+		return nil, fmt.Errorf("unable to unmarshal json for value")
```

Comment:
Would suggest to include Kubernetes in this error, so something like this:

> unable to unmarshal json for value while parsing Kubernetes Identity.

WDYT?